src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
+
+const has_features_2_6 = true;
```

Comment:
AFAICS it is neither a feature nor a bug fix - it is purely infrastructural maintenance for a feature that will be added in v2.7 (this PR). To make `hasFeatureFrom` helper reliable we need to add that module to all supported branches in all core libraries - it has no effect on their own API/functionality.